src/site/static/eligibility.liquid

```diff
@@ -1871,8 +1871,29 @@ pageClass: "page-public-assistance"
   }
 
   function gaEligible() {
-    // TODO
-    return false;
+    // https://socialservices.sccgov.org/financial-assistance/financial-assistance-adults-without-dependent-children-general-assistance
+    const age = document.getElementById("age").value || null;
+    const numDependents = document.querySelectorAll('input[id^="hh-member-dependent"]:checked').length;
+  
+    const MIN_GA_ELIGIBLE_AGE = 18;  // Years
+    const MAX_GA_ELIGIBLE_AGE = 64;  // Years
+    const NUM_OF_DEPENDENTS = 0;     // None
+    const MAX_RESOURCES = 500;       // USD Combined household assets
+    const MAX_GROSS_INCOME = 150;    // USD per month
```

Comment:
It turns out I was wrong about this when I made the Benefits Eligibility spreadsheet.  The max gross income can't exceed the maximum grant level, and the maximum grant level scales with family size and living arrangement.  I'm not exactly sure of the difference between shared housing and unshared housing.  I wonder if we need to add more inputs to the form...

See Section 4.2 [here](https://stgenssa.sccgov.org/debs/policy_handbook_Charts/ch-ga.pdf)

Reply:
Thanks for adding the TODO comment regarding this.